### ISSUE TITLE
Code for model training

### ISSUE THREAD

**Issue by HenriettaHolze** [rosstewart/MutPred-PPI#1]
Hi Ross, 

Exciting to see a new model for mut/ppi prediction! 
I would like to gain a deeper understanding of the model and would therefore be keen to see the training code. 
Best case, how to train the model using the files available on Zenodo (including pre-training). 
Would it be possible for you to add it? 

A minor question about the zenodo data: You write that you "removed duplicate data points, those with conflicting labels, and those with failed structure generation". 
The csv file `sahni_fragoza_train.csv` contains mutations that have conflicting evidence in Sahni and Fragoza (e.g. Q96JS3 P256L Q9UFF2, Q9Y4Z2 F199S F5GY10). 
It also contains duplicated data (P55347-1,V265I,P26583 / P55347,V265I,P26583).
Did you only keep the labels from either Sahni or Fragoza, or did you remove the data points at a later step? 

Best wishes, 
Henrietta

**Comment by rosstewart** [comment]
Hi again Henrietta,

Regarding both of these points: please see the new README and docs for instructions on model training and reproducing analyses (notebooks/reproduce_all_figures.py reproduces all figures and analyses including pretraining, fine-tuning, benchmark predictions, and variant repository analyses end-to-end). As for the data issue, I have remapped all data from the original sources (Sahni et al. 2015 and Fragoza et al. 2019) instead of using pre-mapped data. You can find the new results on Zenodo (https://zenodo.org/records/22774603) and remap them yourself by following notebooks/map_ppi_datasets.py, which goes through all of the deduplication and conflict removal steps. Let me know if you have any questions, I am happy to help.
